assets/js/app.js

```diff
@@ -26,11 +26,11 @@ const app = createApp({
     },
   },
   methods: {
-    raise: function (msg) {
-      console[msg.type](msg);
+    log: function (msg) {
```

Comment:
Eine Log Funktion bei der Nachrichten jetzt immer Fehler sind?

Reply:
Wir können das auch in raise umbenennen. Über diesen Mechanismus haben/werden wir immer nur Fehler und Warnungen ausspielen.